owl/SOMA.owl

```diff
@@ -148,7 +154,25 @@ AnnotationPropertyRange(SOMA:symbol xsd:string)
 #   Object Properties
 ############################
 
-# Object Property: <http://www.ease-crc.org/ont/SOMA.owl#after> (<http://www.ease-crc.org/ont/SOMA.owl#after>)
+# Object Property: <http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> (is initial scene of)
+
+AnnotationAssertion(rdfs:comment <http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> "A relation between StateTransitions and Scenes, which identifies the scene the transition starts from.")
+AnnotationAssertion(rdfs:label <http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> "is initial scene of")
+SubObjectPropertyOf(<http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isObjectIncludedIn>)
+InverseObjectProperties(<http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> SOMA:hasInitialScene)
+ObjectPropertyDomain(<http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> SOMA:Scene)
+ObjectPropertyRange(<http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> SOMA:StateTransition)
+
+# Object Property: <http://www.ease-crc.org/ont/SOMA-SAY.owl#isTerminalSceneOf> (is terminal scene of)
+
+AnnotationAssertion(rdfs:comment <http://www.ease-crc.org/ont/SOMA-SAY.owl#isTerminalSceneOf> "A relation between StateTransitions and Scenes, which identifies the scene the transition is expected to end at.")
+AnnotationAssertion(rdfs:label <http://www.ease-crc.org/ont/SOMA-SAY.owl#isTerminalSceneOf> "is terminal scene of")
+SubObjectPropertyOf(<http://www.ease-crc.org/ont/SOMA-SAY.owl#isTerminalSceneOf> <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isObjectIncludedIn>)
```

Comment:
Same story as for isInitialSceneOf: the semantics of Situation have changed, so assuming Situations are objects is no longer appropriate.